cmd/crc/cmd/root.go

```diff
@@ -38,3 +47,20 @@ func Execute() {
 		os.Exit(1)
 	}
 }
+
+func initViper() {
+	v := viper.New()
+	v.SetConfigFile(constants.ConfigPath)
```

Comment:
where is configpath?

Reply:
`$HOME/.crc.json` for now, till we have the directory structure in place.